Cargo.toml

```diff
@@ -1,10 +1,13 @@
 [package]
 name = "aw_server"
 version = "0.1.0"
-authors = ["Johan Bjäreholt <johan@bjareho.lt>"]
+authors = ["Johan Bjäreholt <johan@bjareho.lt>", "Erik Bjäreholt <erik@bjareho.lt>"]
+
+[target.'cfg(target_os="android")'.dependencies]
```

Comment:
Put platform-specific stuff att bottom of Cargo.toml pls

Reply:
Fixed in https://github.com/ActivityWatch/aw-server-rust/pull/8/commits/c11fab3c22e51fc45358a19906ceb3359e389ecf